### PR TITLE
Create ip button on web

### PR DESCRIPTION
**Pull request by osvegn**
# New feature
Add a text input to write the ip address of the user in front web

image
image

### CODE REVIEW

File: Web/src/AuthPage.js [hunk 1]
```diff
@@ -56,17 +57,6 @@ function AuthPage(props) {
         }
     }
 
-    /* Checking if the user is already logged in. If he is, it redirects him to
-    the home page. */
-    auth.onAuthStateChanged(user => {
```
Comment (31Nathan) [inline Web/src/AuthPage.js]: Get Back

**Review by 31Nathan — CHANGES_REQUESTED** [review]

File: Web/src/AuthPage.js [hunk 2]
```diff
@@ -175,20 +176,87 @@ function AuthPage(props) {
...
+                stravaId: props.userInformation.stravaId
+            },
+            ip: event.target.value
+        })
+    }
+
     /**
      * It returns a form with an email input, a password input, a button to submit
      * the form, and a link to change the authentication mode
      * @returns A form with a title, an email input, a password input, a button to
```
Comment (STom6) [inline Web/src/AuthPage.js:257]: update doc

**Comment by JohanCDev** [comment]
We don't add a button on the settings screen as in the app ?

**Review by STom6 — APPROVED** [review]

**Review by JohanCDev — APPROVED** [review]
lgtm

**Review by 31Nathan — APPROVED** [review]
LGTM

**Pull request merged**